quconot/implementations/mct_parallel_decomposition.py

```diff
@@ -20,8 +20,19 @@ def __init__(self, controls_no: int, **kwargs) -> None:
         pass
 

```

Comment:
no details added for this
this is fig 10a implementation from Efficient constructions for simulating Multi Controlled Quantum Gates